src/snakemake_interface_common/utils.py

```
@@ -2,9 +2,10 @@
 from datetime import datetime
 import os
 import subprocess as sp
+from typing import Any, Callable, Optional, Tuple, Type
```

Ruff (0.8.2)

`typing.Type` imported but unused

Remove unused import: `typing.Type`

(F401)